script/prove-block.sh

```diff
@@ -33,8 +33,10 @@ elif [ "$chain" == "taiko_a7" ]; then
 	l1Rpc="https://ethereum-holesky-rpc.publicnode.com"
 	beaconRpc="https://eth-holesky-beacon.public.blastapi.io"
 else
-	echo "Invalid chain name. Please use 'ethereum', 'holesky' or 'taiko_a7'."
-	exit 1
+	echo "Using customized chain name $1. Please double check the RPCs."
+	rpc="https://rpc.hekla.taiko.xyz"
+	l1Rpc="https://ethereum-holesky-rpc.publicnode.com"
+	beaconRpc="https://eth-holesky-beacon.public.blastapi.io"
```

Comment:
Maybe we can make the RPCs part of the chain spec? Because people can now just go the chain spec config file and change them there if needed. This would also be helpful for the tests that depend on RPC urls because then we can also just load them from the default chain spec configs.

Reply:
could be, I think it makes chainspec super set further, as l2 does not need beacon so far, although it is already a super set of l1+l2.

Reply:
Yeah not sure, but seems okay to me to have config members be useless in some cases, and having this data in a single place that is easily accessible from different places seems nice.

Reply:
Yes, I am doing this, in chain spec still feels better than in proof request.